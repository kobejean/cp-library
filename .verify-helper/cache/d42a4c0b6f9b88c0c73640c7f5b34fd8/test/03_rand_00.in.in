35
32 56
69 64
8 0
19 56
84 62
97 29
1 63
20 35
20 49
66 30
10 44
36 10
56 52
5 43
23 80
25 34
31 76
91 75
32 69
51 11
30 29
43 33
9 90
68 72
60 4
92 49
13 72
44 90
24 90
34 43
47 17
27 51
26 27
93 8
29 62
10
8 58 93 143
88 138 38 88
85 135 93 143
88 138 20 70
11 61 46 96
87 137 10 60
9 59 87 137
68 118 3 53
73 123 0 50
74 124 5 55
