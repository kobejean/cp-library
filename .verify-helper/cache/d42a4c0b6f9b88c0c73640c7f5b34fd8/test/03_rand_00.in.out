
17
25


5
25

0
3
8
12
14
16
18
26
27
28
31
34

5
25

22
27
28

5
25
33

5
25
33

5
25
33

